src/utils/codeRainingEffect.ts

```diff
@@ -0,0 +1,89 @@
+let intervalId: number | null = null;
+let canvas: HTMLCanvasElement | null = null;
+let style: HTMLStyleElement | null = null;
+
+export const startRainingEffect = () => {
+  if (intervalId) return;
+
+  canvas = document.createElement('canvas');
+  canvas.style.opacity = '0';
+  canvas.style.transition = 'opacity 1s ease';
```

Comment:
Here you are using CSS styles. However, on line 95 of `src/views/HomeView.vue`, you again are using transitions. Are both necessary? Perhaps one or the other?